DEVELOPER.md

```diff
@@ -49,286 +13,370 @@ To test and run services, you need to get access to the development Kubernetes c
 Our guide uses Minikube as an example because it is easy to get it started, but you can use another Kubernetes cluster as well.
 Please see [Kubernetes](https://kubernetes.io) for setting up a development cluster.
 
-Please read [doc](https://www.fluvio.io) for a technical arch and operation guide. 
+Please read [doc](https://www.fluvio.io) for a technical arch and operation guide.
 
 ---
 
 ## Setting up Development Environment
 
 The development environment requires the Rust toolchain and a few other build dependencies to build changes to the source code.
 
-To deploy and test the source code, Kubernetes is required because Fluvio stores its metadata in Kubernetes. Fluvio binaries can be tested locally, running outside of any k8s cluster using the `--local` parameter or they can be built into a docker image and deployed into the Kubernetes cluster.
+Fluvio allows you to run the cluster locally or in a Kubernetes cluster, when
+running on a Kubernetes Cluster metadata is stored in Kubernetes, as well as
+system components such as the SC and SPU(s).
 
-### Rust toolchain
+As an alternative, you can run Fluvio locally without Kubernetes, metadata will
+be stored in disk and processes will spawn as system processes.
 
+By default, Fluvio will create a cluster in the local environment unless you
+specify it to run on a Kubernetes cluster.
 
-Please follow [setup](https://www.rust-lang.org/tools/install) instructions to install Rust and Cargo.
+## Requirements
```

Comment:
Why this section is needed anymore?

Reply:
Requirements list dependencies needed to build Rust source code. Its a OS independent section. For now only Rust toolchain is pointed to an installation link, Git and Make are pointed to its official websites.